fly/commands/execute.go

```diff
@@ -127,6 +132,13 @@ func (command *ExecuteCommand) Execute(args []string) error {
 		return err
 	}
 
+	if command.Background {
+		fmt.Printf("%d", build.ID)
+
+		os.Exit(0)
+		return nil
+	}
+
 	fmt.Printf("executing build %d at %s\n", build.ID, clientURL.ResolveReference(buildURL))
```

Comment:
The only thing I want to think about a bit is the output when you run `fly execute` with this flag. Right now it's printing out the build ID because, I assume, it gets you what you want for your use-case.

What if someone wants the full URL to the build though? By not printing the URL I think it becomes harder for users to discover that URL. The current output is not unparsable as well. `fly execute ... | awk '{print $3}'` would easily get you the build ID.

I think we should prefer to keep the current output as-is and just exit early. WDYT?

```suggestion
	fmt.Printf("executing build %d at %s\n", build.ID, clientURL.ResolveReference(buildURL))
	
	if command.Background {
		os.Exit(0)
		return nil
	}
```

Reply:
Agreed, it was really specific 😄 I've changed the test accordingly.

Though someday, I suggest adding an arg like `--output=json` to all commands to dump multiple data from which parent processes can pick. Even in the tests, you rely on a regex to get the build id back. But that may be a big change that you'd better handle.

Reply:
Thanks for the quick turnaround! And totally agree, I had the same thought about having an `--output` for all commands. `fly` itself probably deserves a v2 of or at least some major internal re-working.